core/src/commonMain/kotlin/com/xebia/functional/xef/llm/MemoryManagement.kt

```diff
@@ -42,9 +39,7 @@ internal suspend fun List<CreateChatCompletionResponseChoicesInner>.addChoiceWit
 ): List<CreateChatCompletionResponseChoicesInner> = also {
   val cid = scope.conversationId
   if (history != MessagesToHistory.NONE && isNotEmpty() && cid != null) {
-    val aiMemory =
-      this.filter { it.message.content != null }
```

Comment:
We were deleting messages without `content` and with `toolCalls`, now it is not necessary.